maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -310,6 +343,57 @@ private void downloadOWAs(File targetDirectory, DistroProperties distroPropertie
 		}
 	}
 
+	/**
+	 * Sets the configuration folder for the specified server using the provided distro properties.
+	 *
+	 * @param server           The server for which to set the configuration folder.
+	 * @param distroProperties The distro properties containing the configuration information.
+	 */
+	private void setConfigFolder(Server server, DistroProperties distroProperties) throws MojoExecutionException {
+		if(distroProperties.getConfigArtifacts().isEmpty()) {
+			return;
+		}
+		File configDir = new File(server.getServerDirectory(), SDKConstants.OPENMRS_SERVER_CONFIGURATION);
+		configDir.mkdir();
+		downloadConfigs(distroProperties, configDir);
+		File referenceApplicationFile = new File(configDir, "referenceapplication-distro.owa");
+		if (!referenceApplicationFile.exists()) {
+			return;
+		}
+		try {
+			ZipFile zipFile = new ZipFile(referenceApplicationFile);
+			zipFile.extractAll(configDir.getPath());
+			for (File file : Objects.requireNonNull(configDir.listFiles())) {
+				if (file.getName().equals("openmrs_config")) {
+					FileUtils.copyDirectory(file, configDir);
+				}
+				FileUtils.deleteQuietly(file);
+			}
+			FileUtils.deleteQuietly(referenceApplicationFile);
+		}
+		catch (ZipException | IOException e) {
+			throw new RuntimeException(e);
+		}
+	}
+
+	/**
+	 * Downloads the configuration files specified in the distro properties and saves them in the provided config directory.
```

Comment:
Can you point me to the part of the distro properties file where configuration files are specified? I am looking at an example like this: https://github.com/openmrs/openmrs-distro-referenceapplication/blob/main/distro/distro.properties

Reply:
Configurations that are like `config.referenceapplication-distro=3.0.0-SNAPSHOT`.

Reply:
Would this be the way to specify the version of the reference application distro to use?